### PR TITLE
[15.0][ADD] mail_environment_office365

### PR DESCRIPTION
**Pull request by ajaniszewska-dev**

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]

File: mail_environment_office365/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Camptocamp SA (https://www.camptocamp.com).
+# @author Anna Janiszewska <anna.janiszewska@camptocamp.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Mail configuration with server_environment for Office365",
+    "summary": "Configure Office365 servers with server_environment_files",
```
Comment (simahawk) [inline mail_environment_office365/__manifest__.py]:
```suggestion
    "name": "Server environment configuration for Office365",
    "summary": "Configure Office365 parameters with environment variables via server_environment",
```

**Review by simahawk — APPROVED** [review]
LG

**Comment by simahawk** [comment]
@ajaniszewska-dev pls rewrite the commit msg to remove the odoo version, then you can cherry-pick the same commit here https://github.com/OCA/server-env/pull/119

**Comment by ajaniszewska-dev** [comment]
> @ajaniszewska-dev pls rewrite the commit msg to remove the odoo version, then you can cherry-pick the same commit here #119

@simahawk  not sure i got your point, i did use cherry-pick command, just changed version in manifest and in commit. anyway, did it once again with removing version from commit.

**Comment by simahawk** [comment]
You must cherry-pick the commit and then add a migration commit ;)

**Comment by ajaniszewska-dev** [comment]
@OCA/server-environment-maintainers  can we merge this one?

**Pull request merged**